src/library/utils/isValidProp.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
+
+import _htmlAttributes from 'react-html-attributes';
+import memoize from 'fast-memoize';
+import reactProps from './reactProps';
+import { settify } from './collections';
+
+type HtmlAttributes = {
+  '*'?: Set<string>,
```

Comment:
I didn't know you could do that! That would've come in handy in the `createTheme` work a few PRs back...

Reply:
Or maybe I'm misunderstanding. Is that literally an object property with a key of `'*'`?

Reply:
yes literally a `'*'` key in the object. it's where all the attributes that apply to every DOM element live in that import